### DISCUSSION TITLE
Mooving from deCONZ to Zigbee2Mqtt with a Conbee II

### DISCUSSION THREAD

**Discussion by samsam-rolon** [Koenkk/zigbee2mqtt#6230] [Devices]
Hello
I own a conbee II.
May I can stop my docker deconz, start a container zigbee2mqtt for conbee without re-include all my devices ?
Or I need to re-include every devices ?

I just want to "test" it few days with my whole network.

Regards

**Comment by dircr** [comment]
You need to re-pair, unfortunately.

**Comment by Hedda** [comment]
dresden-elektronik deCONZ developers need to add an option to save backups in the "Open ZigBee Coordinator Backup Format" standard:

Check out the directly related feature request Dresden Elektronik here ->  https://github.com/dresden-elektronik/deconz-rest-plugin-v2/issues/12
